Lectures/Lecture8_OpenScienceReflections.md

```diff
@@ -1,41 +1,97 @@
++ Data structures and data plans are a waste of time.
++ A data-intensive project without a data plan cannot be trusted.
++ Efforts to make articles machine readable is a waste of resources.
++ Forcing authors to make their data available before publication is legitimate (wasting their time).
++ Maintenance of open source projects should (not) be part of scientists' job description.
++ Open-source hardware cannot be trusted for serious scientific work.
++ Open-source hardware is necessary for making science accessible.
++ Open Science should (not) be a mandatory part of our master program.
++ Science communication is part of the job of a journalist, not a scientists
++ Activism hurts the trust in scientists 
++ The most important job/role of scientists is to: ...
++ Technology transfer is a distraction for fundamental research
++ Open science can (never) be a profitable business
++ The real impact of research can (never) be measured
++ Careers in open science are less (more) rewarding than traditional science careers
++ Data stewardship is a part of the researchers' duties 
++ Data stewardship is a service that should be done by librarians
++ Of the five schools of Open Science,I would advocate for
+	+ Public school
+	+ Infrastructure school
+	+ Pragmatic school
+	+ Measurement school
+	+ Democratic school
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ After watching all the videos in the class, each student distributes 100 points between the presented pitches based on how convincing they find that video.
 
-### Assignment description
+## Instructions 
 
++ for creating a pitch
+	1. Begin by outlining the core message or goal of your pitch. Clearly state what issue are you aiming to address and for whom.
+	2. Identify the main features, benefits or unique selling points of your project. Highlight what sets it apart from existing solutions or approaches.
+	3. Structure your pitch in a logical and engaging manner. Consider including an attention-grabbing introduction, a concise explanation of the problem or need, a description of your solution or idea and a compelling conclusion.
+	4. Ensure that your pitch is concise, coherent and easy to understand. Avoid jargon or technical terms that may confuse your audience.
+	5. Once you have finalised the written pitch, take it a step further by visualising the key elements of your idea using illustrations, diagrams or sketches. This will help communicate your concept more effectively.
+	6. Review and refine your written pitch, ensuring they align with the overall message and goals of your project. Seek feedback from your peers or teacher and make necessary adjustments to enhance clarity and impact.
++ for recording and editting a clear video
+	1. location: choose a a quiet environment and avoid distracting backgrounds
+	2. acoustics: pay special attention to sound quality and make sure your recorded voice is clearly and consistently audible
```

Comment:
Are we going to help them by providing microphones? This really was a big issue last year. Can we borrow some microphones from AV or IT?